lib/ii.c

```diff
@@ -4588,13 +4625,19 @@ grn_ii_update_one(grn_ctx *ctx, grn_ii *ii, grn_id tid, grn_ii_updspec *u, grn_h
         pos = a[0];
         if ((pseg = buffer_open(ctx, ii, pos, &bt, &b)) == GRN_II_PSEG_NOT_ASSIGNED) {
           DEFINE_NAME(ii);
+          grn_obj token;
```

Comment:
We should not put it after `DEFINE_NAME` because it has code not variable definitions.

Reply:
It should be:

```c
grn_obj token;
DEFINE_NAME(ii);
```